wgengine/netstack/netstack.go

```diff
@@ -373,7 +372,7 @@ func (ns *Impl) updateIPs(nm *netmap.NetworkMap) {
 		pa := tcpip.ProtocolAddress{
 			AddressWithPrefix: ipp,
 		}
-		if ipp.Address.To4() == "" {
+		if ipp.Address.String() == "" {
```

Comment:
I'd prefer to not allocate here (by calling String) just to test if it's unset

Reply:
Would `ipp.Address.Unspecified()` work?